vm/systemSmartContracts/auction.go

```diff
@@ -412,11 +418,43 @@ func (s *stakingAuctionSC) setConfig(args *vmcommon.ContractCallInput) vmcommon.
 		UnJailPrice:   big.NewInt(0).SetBytes(args.Arguments[5]),
 	}
 
+	if auctionConfig.MinStakeValue.Cmp(zero) <= 0 {
```

Comment:
maybe you can make a method for all these checks

Reply:
fixed